Source/WebKit/UIProcess/API/APIWebAuthenticationPanelClient.h

```diff
@@ -43,9 +43,10 @@ class AuthenticatorAssertionResponse;
 
 namespace API {
 
-class WebAuthenticationPanelClient {
+class WebAuthenticationPanelClient : public RefCounted<WebAuthenticationPanelClient> {
     WTF_MAKE_TZONE_ALLOCATED_INLINE(WebAuthenticationPanelClient);
 public:
+    static Ref<WebAuthenticationPanelClient> create() { return adoptRef(*new WebAuthenticationPanelClient); }
```

Comment:
We need to make the following constructor protected too, otherwise nothing is forcing clients to use `create()`:
```
protected:
    WebAuthenticationPanelClient() = default;
```